### ISSUE TITLE
Notification Squad Applications

### ISSUE THREAD

**Issue by marazmatik** [eveseat/seat#857]
* **Problem**:  Notification alert "Squad Applications" doesnt sent any notification about rewuests to squads.
But it sends if shoose  "New Squad Member" Alert.

* **Expected**: Added new integration to 
* **Logs / Screenshots / Proof**: 
![image](https://user-images.githubusercontent.com/2928910/151660059-80a3bdf2-6da1-432d-a7bf-9e1f9d460eb5.png)
* **Version Info**: 
* ```
| Vendor   | Package Name             | Installed Version        |
| -------- | ------------------------ | ------------------------ |
| eveseat  | api                      | 4.7.0                    |
| eveseat  | console                  | 4.8.0                    |
| eveseat  | eveapi                   | 4.16.0                   |
| eveseat  | notifications            | 4.3.1                    |
| eveseat  | services                 | 4.2.0                    |
| eveseat  | web                      | 4.17.0                   |
| denngarr | seat-fitting             | 4.0.5                    |
| denngarr | seat-srp                 | 4.2.6                    |
| humunuk  | alliance-structure-mngmt | 0.5.2                    |
| kassie   | calendar                 | 4.0.6                    |
| ccp      | eve_online_sde           | sde-20211207-TRANQUILITY |
```

**Comment by warlof** [comment]
is your squad manual or don't have any moderators assigned ?

**Comment by marazmatik** [comment]
Its manual and moderated, if in notification i have "New Squad Member", then i become to  recieve notifications "A SeAT Squad has a new Application!:
![image](https://user-images.githubusercontent.com/2928910/151666873-65f7a766-7dfc-4fb1-a66e-7482146fbdb0.png)

**Comment by warlof** [comment]
then it's normal, your squad don't relay on approval (it's manual, meaning you assign user to the squad).

**Comment by marazmatik** [comment]
New Squad Member -  This notification is when new member added to squad
![image](https://user-images.githubusercontent.com/2928910/151670867-152708c2-d94c-4a51-b29d-95392128919c.png)

Squad Applications - when condidate ask for enter squad
![image](https://user-images.githubusercontent.com/2928910/151670895-b051a0b4-3b47-4c71-8b08-01cacf41285f.png)

Am i wrong?

**Comment by warlof** [comment]
Correct, but manual squad do not relay on approval if no moderators is attached to the squad. In such case, user is automatically becoming a member and bypass approval flow.

**Comment by marazmatik** [comment]
i Have squad, manual , moderated, with moderator, in notifications i have only Squad Applications,
Candidate asking to join, and i have no notification
![image](https://user-images.githubusercontent.com/2928910/151720876-d155a102-0a41-46fa-9eb3-be0cd1ee402e.png)

**Comment by warlof** [comment]
Right, that's not normal. From where did you get the squad application notification screen ?

**Comment by marazmatik** [comment]
its start to notify when i add "New Squad Member" in notifications
